packages/pn-personafisica-webapp/src/components/Contacts/__test__/AddSpecialContactDialog.test.tsx

```diff
@@ -61,10 +61,8 @@ describe('test AddSpecialContactDialog', () => {
     expect(titleEl).toHaveTextContent(`special-contacts.modal-title`);
     const bodyEl = within(dialog).getByTestId('dialog-content');
     expect(bodyEl).toBeInTheDocument();
-    expect(bodyEl).toHaveTextContent(`special-contacts.pec`);
-    const input = getById(bodyEl, 's_value');
```

Comment:
restore this code and use queryById and check that the input is not in the document